senders/types.go

```diff
@@ -58,3 +41,7 @@ type EventSender interface {
 	// Sends an event to Wavefront with optional tags
 	SendEvent(name string, startMillis, endMillis int64, source string, tags map[string]string, setters ...event.Option) error
 }
+
+type SpanTag = span.Tag
```

Comment:
SpanTag and SpanLog are types used in public interfaces of the senders package.  Does it make sense to declare them as equivalent to internal types?  I am not yet sure how godoc will handle this, but if I were learning this API, I'd want to see the full definition of SpanTag and SpanLog just because they are part of public interfaces.

Reply:
Yeah, that's the only thing that was 'hard' about this set of changes, because having the internal package depend on the senders package created a circular dependency. I agree we do want the full definition in godoc. I'll try rendering the godoc locally and see whether we get the full type definition there.

There's definitely some kind of smell happening. I want the `senders` package to own all the public apis, and most of the line handlers only depend on primitive types. Spans are the only ones where the line formatter wants to use a custom struct type.

The struts are very simple so I suppose a translation from `SpanTag` to `span.Tag` would be easy to write, with both structs having the same fields.

Reply:
Yeah, you should be able to move between SpanTag and span.Tag using a simple type conversion.